var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -154,16 +154,21 @@ def run_cxx_example_test(self):
 
         test_dir = join_path(self.test_suite.current_test_cache_dir, 'examples', 'apps')
 
-        if not os.path.exists(test_dir):
-            print('Skipping caliper test')
+        if not os.path.isfile(join_path(test_dir, 'cxx-example.cpp')):
+            print('Skipping caliper test: file does not exist')
             return
 
         exe = 'cxx-example'
 
+        if os.path.exists(self.prefix.lib):
+            lib_dir = self.prefix.lib
+        else:
+            lib_dir = self.prefix.lib64
```

Comment:
Is there any chance this won't exist either?

Reply:
The caliper docs say there should be some kind of lib dir there.